piptools/utils.py

```diff
@@ -522,3 +525,104 @@ def parse_requirements_from_wheel_metadata(
             markers=parts.markers,
             extras=parts.extras,
         )
+
+
+def pyproject_toml_defaults_cb(
```

Comment:
This needs to be renamed for the same reason. It is inaccurate that this only reads `pyproject.toml`.
Also, since it's a function, its name should start with a verb describing the action it performs, not a contextless static state.

Reply:
Marking this thread as resolved.